library/std/src/lazy.rs

```diff
@@ -293,7 +293,7 @@ impl<T> SyncOnceCell<T> {
 
         // Don't drop this `SyncOnceCell`. We just moved out one of the fields, but didn't set
         // the state to uninitialized.
-        mem::ManuallyDrop::new(self);
+        mem::forget(self);
```

Comment:
FYI @matklad you may want to upstream this change to `once_cell`.

Reply:
If you want to be notified about changes to `lazy.rs` you can add yourself to highfive list: https://github.com/rust-lang/highfive/blob/master/highfive/configs/rust-lang/rust.json